modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp

```diff
@@ -33,6 +34,10 @@
         if(usePKCE) {
             session.setAttribute(OAuth2Constants.OAUTH2_USE_PKCE, usePKCE);
         }
+        boolean formPostMode = formPostParameter != null && "yes".equals(formPostParameter);
```

Comment:
Shall we define a private constant for "yes" here

Reply:
Fix with https://github.com/wso2/product-is/pull/1369